### PR TITLE
Check arguments individually in `brew cask list`.

### PR DESCRIPTION
**Pull request by reitermarkus**
### Changes to the core
- [x] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

Fixes part 1 of https://github.com/caskroom/homebrew-cask/issues/14058.

### CODE REVIEW

File: lib/hbc/source.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def self.sources
 
   def self.for_query(query)
     odebug "Translating '#{query}' into a valid Cask source"
+    raise Hbc::CaskUnavailableError.new(query) if query.to_s =~ /^\s*$/
```
Comment (claui) [inline lib/hbc/source.rb]: This breaks `brew cask list` for me:

image

**Comment by claui** [comment]
Thanks for your contribution @reitermarkus! Looks good to me :+1:

**Comment by reitermarkus** [comment]
You're welcome! 😃

**Comment by vitorgalvao** [comment]
@claui Feel free to merge these when you’re happy with the submissions (not sure if you’d like to wait for more people to look at it and comment, but I’m fine with it).

**Pull request merged**